src/app/core/config/config.service.ts

```diff
@@ -377,7 +380,72 @@ const migrateUserEntityAndPanels: ConfigMigration = (key, configPart) => {
 };
 
 /**
- * Remove outdated task view configs
+ * Migration to set or update entityType for specific components
+ */
+const migrateComponentEntityTypeDefaults: ConfigMigration = (
+  key,
+  configPart,
+) => {
+  if (typeof configPart !== "object" || !configPart?.component) {
+    return configPart;
+  }
+
+  if (!configPart.config) {
+    configPart.config = {};
+  }
+
+  const defaults = ENTITY_DEFAULT_VALUES[configPart.component];
```

Comment:
Maybe `RELATED_ENTITIES_COMPONENT_DEFAULT_CONFIGS` (or something similar but a bit shorter) would be more explicit?